ui/src/app/utils/title.ts

```diff
@@ -0,0 +1,14 @@
+import {environment} from 'environments/environment';
+
+import {BASE_TITLE} from 'app/utils/strings';
+
+export function buildPageTitleForEnvironment(pageTitle?: string) {
+  let title = BASE_TITLE;
```

Comment:
One possiblity to avoid the `let`:
```
    const inner = environment.shouldShowDisplayTag && pageTitle ? `${pageTitle} | ` : ''
    return environment.shouldShowDisplayTag ? `${inner}[${environment.displayTag}] ${BASE_TITLE}` : BASE_TITLE
```
But I don't know if that is cleaner to read

Reply:
That's exactly why I did it as a `let`. I don't think mutables are so evil as to make it better to do something this much harder to parse